.jitpack.yml

```
@@ -6,3 +6,4 @@ before_install:
    - wget http://central.maven.org/maven2/org/scala-sbt/sbt-launch/1.2.8/sbt-launch-1.2.8.jar
 install:
    - java -jar sbt-launch-1.2.8.jar +publishM2
+   - SCALAJS_VERSION="1.1.0" ; java -jar sbt-launch-1.2.8.jar +publishM2
```

I mean, without the semicolon. Else, the local variable is not picked up by the java command.

```suggestion
   - SCALAJS_VERSION="1.1.0" java -jar sbt-launch-1.2.8.jar +publishM2
```